### DISCUSSION TITLE
Material You Colors

### DISCUSSION THREAD

**Discussion by MuhammadBayiz** [trakt/showly#723] [Ideas / Feature Requests]
Would you make the app follow Android 12+ material you colors?
Currently the only follows material design not material colors

**Comment by Xtrems876** [comment]
Second this. This is pretty much the only feature that entices me towards other open source apps of this kind, as I do have a preference for consistency across my system.
On the other hand, Showly does have a pretty specific look and feel that distinguishes it from the rest. The black-and-red colors that remind you of cinemas. The question is how to preserve that while also following androids design language and color scheme.
